### PR TITLE
[AIRFLOW-3988] Airflow CLI - Sort commands alphabetically

### PR DESCRIPTION
**Pull request by cchung100m**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-3988
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.
  - In case you are proposing a fundamental code change, you need to create an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)).
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).

### Description

- [ ] Here are some details about my PR, including screenshots of any UI changes:

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
  - If you implement backwards incompatible changes, please leave a note in the [Updating.md](https://github.com/apache/airflow/blob/master/UPDATING.md) so we can assign it to a appropriate release

### Code Quality

- [ ] Passes `flake8`

### CODE REVIEW

**Comment by cchung100m** [comment]
Hi @ashb @XD-DENG 

PTAL, thanks

File: airflow/bin/cli.py [hunk 1]
```diff
@@ -2395,7 +2395,8 @@ def get_parser(cls, dag_parser=False):
         subparsers.required = True
 
         subparser_list = cls.dag_subparsers if dag_parser else cls.subparsers_dict.keys()
-        for sub in subparser_list:
+        sorted_subparser_list = sorted(subparser_list)
+        for sub in sorted_subparser_list:
```
Comment (XD-DENG) [inline airflow/bin/cli.py]: Thanks @cchung100m , a nice change to me.

But not sure if we need an intermediate variable `sorted_subparser_list` here. Can you consider directly changing `for sub in subparser_list:` into `for sub in sorted(subparser_list):`? Personally I think it's good enough and a bit more concise.

**Review by XD-DENG — CHANGES_REQUESTED** [review]
Thanks @cchung100m . A minor change I would like to suggest

**Comment by XD-DENG** [comment]
Hi @cchung100m , a few suggestions outside the code change scope:
- please provide meaning PR subject so that folks can quickly identify if it's something they can share inputs (I have updated subject of this PR to a meaning one). 
- next time please update your PR details accordingly based on the template, e.g., providing the correct JIRA Link, and provide necessary description.

Thanks again for your PR thought ;-)

**Comment by ashb** [comment]
You can make our lives as committers easier by actually filling out the pull request template

**Comment by cchung100m** [comment]
Hi @ashb @XD-DENG 

Sorry for any inconvenience and thanks for the suggestions :)

**Pull request merged**